libraries/libopae-c/pluginmgr.c

```diff
@@ -281,14 +281,22 @@ STATIC int opae_plugin_mgr_detect_platforms(bool with_ase)
 
 	if (with_ase) {
 		opae_pci_device ase_pf = {
-			.name = "ase",
+			.name = "ase_pf",
```

Comment:
We have existing tests that look for a single device and then check the vendor_id/device_id for 0x8086/0xa5e. OPAE is, unfortunately, returning device_id 0xa5f first. The simulation works, but the SW thinks it isn't ASE.

Would this new code still work if you made the device_id always 0xa5e and varied subsystem-device_id instead?

Reply:
We can give it a try.  I can't think of a reason OTTOMH why it wouldn't.  It will involve a change here and a change in opae_ase.cfg.  Let me know.